lib/wordnet/synset.rb

```diff
@@ -100,13 +198,18 @@ def hypernym
       relation(HYPERNYM)[0]
     end
 
+    # Get the parent synset (higher-level category, i.e. fruit -> reproductive_structure).
+    def hypernyms
+      relation(HYPERNYM)
+    end
+
     # Get the child synset(s) (i.e., lower-level categories, i.e. fruit -> edible_fruit)
```

Comment:
Yeah, pretty much all of this is borked: https://github.com/doches/rwordnet/commit/64869a4fa7b4de60e2a34806e5d0f1ecb967ac7c

Teach me to accept PRs without going through 'em with a fine-toothed comb.